lib/file-operations.js

```diff
@@ -66,6 +66,48 @@ function getTimesDiff(fsStat, vinylStat) {
   return timesDiff;
 }
 
+function getOwnerDiff(fsStat, vinylStat) {
+  if (/^win/.test(process.platform)) {
+    return; // Always return undefined on windows.
+  }
+
+  if (typeof vinylStat.uid !== 'number' &&
```

Comment:
this should probably be abstracted to a method called `isValidUid()` incase we need to add extra logic and it keeps it consistent with `getTimesDiff`